e2e/main.e2e.spec.ts

```diff
@@ -53,32 +53,39 @@ describe('Rancher Desktop', () => {
   });
 
   it('should display welcome message in general tab', async() => {
-    const generalPage: GeneralPage = await navBarPage.getGeneralPage();
+    const generalPage = await navBarPage.getGeneralPage();
 
-    expect(await generalPage.getTitle()).toBe('Welcome to Rancher Desktop');
+    expect(generalPage).not.toBeNull();
+    expect(await generalPage?.getTitle()).toBe('Welcome to Rancher Desktop');
```

Comment:
GitHub seemed to have lost a comment from Eric here…
The `?` is here, because while we know it can't be non-null, TypeScript doesn't (because the assertion doesn't actually tell TypeScript that it's an assertion).

Reply:
yup